rosetta-source/src/main/resources/cdm-sample-files/fpml-5-13/processes/execution-advice/expectations.json

```diff
@@ -2,18 +2,18 @@
 {"fileName":"cdm-sample-files/fpml-5-13/processes/execution-advice/msg-ex51-execution-advice-trade-initiation-C01-00.xml","excludedPaths":0,"externalPaths":63,"outstandingMappings":7,"validationFailures":10,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"CreditDefaultSwap_SingleName","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
 {"fileName":"cdm-sample-files/fpml-5-13/processes/execution-advice/msg-ex52-execution-advice-trade-partial-novation-C02-00.xml","excludedPaths":0,"externalPaths":94,"outstandingMappings":20,"validationFailures":6,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"CreditDefaultSwap_SingleName","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
 {"fileName":"cdm-sample-files/fpml-5-13/processes/execution-advice/msg-ex53-execution-advice-trade-partial-novation-correction-C02-10.xml","excludedPaths":0,"externalPaths":94,"outstandingMappings":20,"validationFailures":6,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"CreditDefaultSwap_SingleName","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
-{"fileName":"cdm-sample-files/fpml-5-13/processes/execution-advice/msg-ex54-execution-advice-trade-partial-termination-C11-00.xml","excludedPaths":0,"externalPaths":55,"outstandingMappings":22,"validationFailures":5,"qualificationExpectation":{"success":true,"qualifyResults":[],"qualifiableObjectCount":0}},
```

Comment:
These FpML execution advice samples do not have a product specified, so no product or tradeLots can be mapped.  Should these samples be removed?